RatFinder/app/src/main/res/layout/content_input_sighting.xml

```diff
@@ -189,7 +189,7 @@
         app:layout_constraintTop_toTopOf="parent" />
 
     <Button
-        android:id="@+id/register"
+        android:id="@+id/cancelsight_id"
```

Comment:
the cancel sighting and create sighting buttons are on top of each other

Reply:
will fix